pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -134,8 +134,7 @@ func Pod(source *v1alpha1.CatalogSource, name string, image string, saName strin
 					ReadinessProbe: &v1.Probe{
 						Handler: v1.Handler{
 							Exec: &v1.ExecAction{
-								Command: []string{"grpc_health_probe", "-addr=:50051"},
-							},
+								Command: []string{"grpc_health_probe", "-addr=:50051"}},
```

Comment:
nit: the last right-bracket on this line matches `ExecAction{` -- is this change a typo?

Reply:
might have come from goland format or make format. I'll :eyes:

Reply:
I think that's correct - it may have been just a format change.